source/_components/media_player.dunehd.markdown

```diff
@@ -26,8 +26,19 @@ media_player:
   - platform: dunehd
     host: IP_ADDRESS
 ```
-Configuration variables:
 
-- **host** (*Required*): IP address or hostname of the device. Example: 192.168.1.32
-- **name** (*Optional*): Name of the device.
-- **sources** (*Optional*): A name-value dictionary of sources that HA component can request to play.
+{% configuration %}
+host:
+  description: IP or hostname of the device. e.g., 192.168.1.32.
```

Comment:
Instead of getting rid of the colon you can put the description between double quotes.
As it is now you have a punctuation error.